troubadix/standalone_plugins/deprecate_vts.py

```diff
@@ -0,0 +1,265 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+import os
+import re
+import sys
+from argparse import ArgumentParser, Namespace
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Iterable, Optional
+
+from troubadix.argparser import file_type
+from troubadix.helper.patterns import (
+    get_special_script_tag_pattern,
+    get_script_tag_pattern,
+    ScriptTag,
+    SpecialScriptTag,
+)
+
+
+class PathException(Exception):
+    pass
+
+
+@dataclass
+class DeprecatedFile:
+    name: str
+    full_path: Path
+    content: str
+
+
+# CHANGE AS NEEDED
+FILENAME_REGEX = re.compile(r"gb_")
```

Comment:
As mentioned in a now "Outdated" comment:

Wouldn't it make sense to use a command line argument for this case instead so that the file itself doesn't need to be touched?